R/na.R

```diff
@@ -33,22 +35,37 @@
 #' levels(f3)
 #' as.integer(f3)
 #' is.na(f3)
-fct_na_value_to_level <- function(f) {
+fct_na_value_to_level <- function(f, level = NA) {
+  if (!identical(level, NA) && !is_string(level)) {
+    cli::cli_abort(
+      "{.arg level} must be a string or {.code NA}, not {.obj_type_friendly level}."
```

Comment:
I think you need `{.obj_type_friendly {level}}`

It isn't letting me do a "suggestion" for some reason

Reply:
Doh